xmlsitemap_menu/xmlsitemap_menu.module

```diff
@@ -68,6 +68,14 @@ function xmlsitemap_menu_entity_info_alter(&$info) {
         'process callback' => 'xmlsitemap_menu_xmlsitemap_process_menu_links',
       ),
     );
+
+    // Change menu edit link to got to configure instead of list.
+    foreach (menu_get_menus() as $type => $name) {
+      if (isset($info['menu_link']['bundles'][$type])) {
+        $info['menu_link']['bundles'][$type]['admin']['path'] = 'admin/structure/menu/manage/%menu/configure';
+        $info['menu_link']['bundles'][$type]['admin']['real path'] = 'admin/structure/menu/manage/' . $type . '/configure';
+      }
+    }
```

Comment:
Should this section be inside the `else {` statement above? It doesn't seem like it would be necessary when defining the new entity type, since that was already done in the first part of the `if {` statement.

However, I don't think this section is needed at all. If another module defined an entity type for menu items, we should just let it use whatever path structure it defined, not override it with new path settings.